src/main/scala/org/globalforestwatch/summarystats/SummaryRDD.scala

```diff
@@ -5,6 +5,7 @@ import com.typesafe.scalalogging.LazyLogging
 import geotrellis.raster._
 import geotrellis.raster.rasterize.Rasterizer
 import geotrellis.spark.SpatialKey
+import geotrellis.spark.partition.SpacePartitioner
```

Comment:
Codacy found an issue: [Imports should be sorted alphabetically](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=5902460)